pandas/core/base.py

```diff
@@ -623,7 +634,8 @@ def __unicode__(self):
 
 
 class IndexOpsMixin(object):
-    """ common ops mixin to support a unified inteface / docs for Series / Index """
+    """ common ops mixin to support a unified inteface / docs for Series /
+    Index """
```

Comment:
PEP8 says that if docstrings are not one-liners, the closing `"""` should be on its own line